mono/mini/mini-amd64.h

```diff
@@ -476,6 +476,10 @@ typedef struct {
             MONO_EMIT_NEW_COND_EXC (cfg, LE_UN, "IndexOutOfRangeException"); \
        } while (0)
 
+// Does the ABI have a volatile non-parameter register, so tailcall
+// can pass context to generics or interfaces?
+#define MONO_ARCH_HAVE_VOLATILE_NON_PARAM_REGISTER 1
+
```

Comment:
This is ok, I'd have preferred simply MONO_ARCH_HAVE_GSHARED_TAILCALLS or something, 'have non volatile non param register' looks way to generic.

Reply:
I believe this covers more than just GSHARED_TAILCALLS, which is why I named it this way. It will cover tailcalls to interface methods too.

Reply:
Ok.